courses/management/commands/manage_topics.py

```diff
@@ -0,0 +1,278 @@
+"""Management command to create topics from a CSV into the system OR assign topics to courses
+
+## Arguments:
+* --file <filename> - Path to the CSV file containing the topics OR the file containing the topics and courses titles (But only one at a time)
+* --create-topics <filename> - Flag to decide that we should create topics from topics sheet (Format mentioned below)
+* --assign-topics <filename> - Flag to decide that we should only populate topics from course title & topics sheet (Format mentioned below)
```

Comment:
I have noticed that we don't need <filename> with these flags as they are boolean flags.